continuous_integration/recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ build:
 
 requirements:
   build:
-    - {{ compiler('rust') }}
+    - rust>=1.62.1
```

Comment:
Just noting like we said internally that we are waiting on https://github.com/conda-forge/rust-activation-feedstock/pull/22 to switch back to using `{{ compiler('rust') }}>=1.62.1`, as the stable platform-specific Rust package currently only goes up to 1.61.0.

cc @jakirkham